app/controllers/admin/reservations/item_pool_searches_controller.rb

```diff
@@ -0,0 +1,23 @@
+module Admin
+  module Reservations
+    class ItemPoolSearchesController < BaseController
+      include Pagy::Backend
+
+      def show
+        item_pool_scope = ItemPool.all
+
+        if params[:category]
+          @category = CategoryNode.where(id: params[:category]).first
```

Comment:
```suggestion
          @category = CategoryNode.find_by(id: params[:category])
```
I think this syntax is preferred but it honestly makes no difference